### ISSUE TITLE
NPE (EMF-CCfinder)

### ISSUE THREAD

**Issue by dmazinanian** [tsantalis/JDeodorant#42]
Clone # 23 (Group 303, Pair 1-2): org.eclipse.emf.ecore.presentation.EcoreEditor#init (row 58) and org.eclipse.emf.mapping.ecore2xml.presentation.Ecore2XMLEditor#init (row 59)

```
java.lang.NullPointerException
    at gr.uom.java.jdeodorant.refactoring.manipulators.RefactoringUtility.getSimpleTypeBindings(RefactoringUtility.java:181)
    at gr.uom.java.jdeodorant.refactoring.manipulators.ExtractCloneRefactoring.pullUpLocallyAccessedFields(ExtractCloneRefactoring.java:2189)
    at gr.uom.java.jdeodorant.refactoring.manipulators.ExtractCloneRefactoring.extractClone(ExtractCloneRefactoring.java:848)
    at gr.uom.java.jdeodorant.refactoring.manipulators.ExtractCloneRefactoring.apply(ExtractCloneRefactoring.java:335)
    at gr.uom.java.jdeodorant.refactoring.manipulators.ExtractCloneRefactoring.checkFinalConditions(ExtractCloneRefactoring.java:4947)
    at ca.concordia.jdeodorant.eclipse.commandline.Application.testRefactoring(Application.java:669)
    at ca.concordia.jdeodorant.eclipse.commandline.Application.start(Application.java:185)
    at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:673)
    at org.eclipse.equinox.launcher.Main.basicRun(Main.java:610)
    at org.eclipse.equinox.launcher.Main.run(Main.java:1519)
    at org.eclipse.equinox.launcher.Main.main(Main.java:1492)
```

**Comment by dmazinanian** [comment]
Same in Hibernate-NiCad (Group 61, Pair 1-2): org.hibernate.test.dialect.functional.cache.SQLFunctionsInterSystemsTest#testSqlFunctionAsAlias (row 140) and org.hibernate.test.legacy.SQLFunctionsTest#testSqlFunctionAsAlias (row 141)

**Comment by tsantalis** [comment]
Fixed in commit [ad573053435b5c0fcb5b07182479498b577db439](https://github.com/tsantalis/JDeodorant/commit/ad573053435b5c0fcb5b07182479498b577db439)
